src/nixpacks/builder/docker.rs

```diff
@@ -222,7 +276,7 @@ impl DockerBuilder {
         };
 
         // -- Setup
-        let mut setup_files: Vec<String> = vec!["environment.nix".to_string()];
+        let mut setup_files: Vec<String> = vec![environment_nix_path];
```

Comment:
I think this can just use `ENVIRONMENT_NIX` instead of `environment_nix_path`

Reply:
Here, we need the relative path `.nixpacks/environment.nix` thats why we need to compute it first. because ENVIRONMENT_NIX only represents `environment.nix`